AugurProject/augur
# Add wallet connect

# Description

Adds [WalletConnect](https://walletconnect.org/) as a login type, allowing [these Wallets](https://walletconnect.org/wallets) to directly connect to Augur.

# Testing
1. On a phone, go to `test.walletconnect.org`.
2. Select `Kovan` or your current network.
3. Select `Connect to WalletConnect`.
4. When the QR code appears, select `Scan` on the WalletConnect test account.

# TODO
- [ ] Get WalletConnect Icon
- [ ] Figure out `getSigners` missing function on provider. This fails when creating the account object.
- [ ] Correctly cleanup. After logging out, you cannot log into a different login type.

# DOD
- [ ] Assert users can log in using WalletConnect.
- [ ] Assert users can log out using WalletConnect.
- [ ] Assert users can log back into WalletConnect with a new account (Assert cleanup).
- [ ] Assert users can log into a separate Wallet type after logging out of WalletConnect (Assert cleanup).
